### DISCUSSION TITLE
More features (filter files / selecting files / remember columns)

### DISCUSSION THREAD

**Discussion by AASabancev** [electerm/electerm#3606] [Ideas]
Hello!

It would be great to have the following features:

1. select a folder/file when typing its name from the keyboard

2. input for filtering files and folders (similar to ls -la | grep filename (with a timeout after input - if you type slowly / or after pressing enter)

3. remember the width of columns when changing (for all connections at once / do not divide your settings for each connection). Otherwise, after reconnecting to the server, the width is reset each time.

thanks!

**Comment by zxdong262** [comment]
#3608
